src/main/java/org/apache/maven/plugins/dependency/utils/DependencyUtil.java

```diff
@@ -226,7 +229,19 @@ public static synchronized void write(String string, File file, boolean append,
             throws IOException {
         Files.createDirectories(file.getParentFile().toPath());
 
-        try (Writer writer = Files.newBufferedWriter(file.toPath(), Charset.forName(encoding))) {
+        OpenOption appendOption;
```

Comment:
Why not make it compact with a ternary operator?

Reply:
can be a ternary operator 😄